### ISSUE TITLE
Help/Hints how to get Script to work

### ISSUE THREAD

**Issue by mburger82** [Geneticus/SEHI-API-Script-Example#1]
Hi

I forked your SEHI dll project and your API-Script Example.
SEHI.dll was compiled successfully. I placed that in my SE-Bin64 folder.
Then added the SEHIScript/data/scripts/SerialVars.cs to the Mod-Folder in %appdata%/SE/Mods...
I can find the local mod in the Mod-Selection. 
When I load the World, it throws some errors when trying to load SEHI-Script.
```javascript
"Compilation of ...\AppData\Roaming\SpaceEngineers\Mods\SEHIScript_SerialVars.cs failed:"
```
and something like
```javascript
"...\AppData\Roaming\SpaceEngineers\Mods\SEHIScript\Data\Scripts\SerialVars.cs(8,6): Error: The type or namespace name 'SEHI' could not be found (are you missing a using directive or an assembly reference?)"
```
It seams the Namespace **SEHI** does not exist. I tried several other namespaces like **VRage.Game.ModAPI**.
My understanding of C# would tell me, that according to the SEHI.dll the namespace of the needed class **MySerialOut** should be **VRage.Game.ModAPI**

Is there something I have to do, to get SE to load **SEHI.dll**?

Would appreciate to get some hints from you.
thx
Martin

**Comment by Geneticus** [comment]
Yeah there’s some issues there. Originally this was slated to go into vanilla, but then the git was broken more often than it worked. This was a quick attempt to hack it in as a plug-in.
